### DISCUSSION TITLE
When will support for Hyper-SD be available?

### DISCUSSION THREAD

**Discussion by supwang** [Acly/krita-ai-diffusion#667] [General]
When will support for Hyper-SD be available?

**Comment by rbfbatx** [comment]
It is available now. I have been using it since it was released.

**Reply by supwang** [reply]
Nice. Could send a screen print of the settings to me? The images that I generated have some problems. Thank you.

**Reply by rbfbatx** [reply]
I am Euler A at 8 steps (w/ the 8 step lora) with 1 CFG. 

![image](https://github.com/Acly/krita-ai-diffusion/assets/156967200/7e843f77-3cf2-4219-a502-bd8ddbe0a67c)

**Reply by supwang** [reply]
Thank you.
I gave it a try, and it seems that both the quality and speed are not as good as ComfyUI's workflow. Looking forward to improvements in the future.
